src/main/scala/mesosphere/marathon/api/v2/PodsResource.scala

```diff
@@ -1,22 +1,41 @@
 package mesosphere.marathon.api.v2
 
+import java.net.URI
 import javax.inject.Inject
 import javax.servlet.http.HttpServletRequest
 import javax.ws.rs._
 import javax.ws.rs.core.{ Context, MediaType, Response }
 
+import akka.event.EventStream
 import com.codahale.metrics.annotation.Timed
+import com.wix.accord.Validator
+import com.wix.accord.dsl._
 import mesosphere.marathon.MarathonConf
+import mesosphere.marathon.api.v2.validation.PodsValidation
 import mesosphere.marathon.api.{ AuthResource, MarathonMediaType, RestResource }
-import mesosphere.marathon.plugin.auth.{ Authenticator, Authorizer }
+import mesosphere.marathon.core.base.Clock
+import mesosphere.marathon.core.event._
+import mesosphere.marathon.core.pod.PodDefinition
+import mesosphere.marathon.plugin.auth._
+import mesosphere.marathon.raml.{ Network, PodDef, PodStatus }
+import spray.json._
```

Comment:
Why are we use spray json here? 
I see 2 issues:
- we should try to have the same error reporting format in v2 
- having 2 json libraries in the same project is harder to deal with